lib/helpers/UiBaseLocators.ts

```diff
@@ -195,7 +195,7 @@ export class UiBaseLocators {
   }
 
   async clickActionsMenuForName(name: string) {
-    await this.page.getByLabel(name).click();
+    await this.page.locator('[label="' + name + '"]').click();
```

Comment:
Did the getByLabel not work?

Reply:
It doesn't work with Content action menu.